src/rules/game.js

```diff
@@ -0,0 +1,17 @@
+import { extractNames } from '../pathUtils'
+
+export const context = 'game'
+export const description = 'Workout game must exit in database or as new files.'
+
+export default function ({data: {game}, type}, {db, otherFiles, team}) {
+  if (type !== 'workout') return true
+  if (typeof game === 'undefined') return true
+  let found = false
+  if (otherFiles && otherFiles.length) {
+    found = otherFiles
+      .filter((f) => f.type === 'insight')
+      .map(({filename}) => extractNames(filename, team))
+      .find(fileSlug => fileSlug === game)
+  }
+  return found || db.insight.count({slug: game})
```

Comment:
Not a big deal but maybe use `some` instead of `find` here, since the check is just for existence anyways?